accessibility_monitoring_platform/apps/common/migrations/0021_populate_issue_numbers.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 5.0.4 on 2024-04-17 08:16
+
+from django.db import migrations
+
+
+def populate_issue_numbers(apps, schema_editor):  # pylint: disable=unused-argument
+    IssueReport = apps.get_model("common", "IssueReport")
+    for issue_report in IssueReport.objects.all():
+        issue_report.issue_number = issue_report.id
+        issue_report.save()
```

Comment:
Are issue ids user facing? If not, Im less concerned that these are 'fixed'

Reply:
The issue report ids are user facing: They appear in the admin when we go through the user feedback and in my email to the users confirming that their feedback has been received.

However, they have no semantic meaning. They merely need to be unique.

Reply:
I just saw that haha